TodoApp/TodoApp.Api/App_Start/JsonFormatterConfig.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Formatting;
+using System.Net.Http.Headers;
+using System.Web;
+using System.Web.Http;
+using Newtonsoft.Json.Serialization;
+
+namespace TodoApp.Api
+{
+    public static class JsonFormatterConfig
+    {
+        public static void Register(HttpConfiguration config)
+        {
+            // Configure Json Formatter
+            JsonMediaTypeFormatter jsonFormatter = config.Formatters.JsonFormatter;
+            jsonFormatter.SupportedMediaTypes.Add(new MediaTypeHeaderValue("text/html"));
```

Comment:
You don't need other than default mediatypes (json). Do you have some specific reason why you want to allow passing data to REST API in another than JSON format?